lib/ansible/plugins/connection/ssh.py

```diff
@@ -461,11 +511,12 @@ class Connection(ConnectionBase):
     def __init__(self, *args, **kwargs):
         super(Connection, self).__init__(*args, **kwargs)
 
+        # TODO: all should come from get_option(), but not might be set at this point yet
         self.host = self._play_context.remote_addr
         self.port = self._play_context.port
         self.user = self._play_context.remote_user
```

Comment:
self.user is being set below now, so is this needed?

Reply:
i tried removing before and failed, but willing to give it a go once i migrate remote_addr completely